tf-modules/archive/db_indexer.tf

```diff
@@ -47,17 +46,6 @@ resource "aws_lambda_function" "db_indexer" {
   }
 }
 
-data "aws_dynamodb_table" "collections" {
```

Comment:
My thought is that we can repeat this process one data type at a time - moving the logic for updating ES into the API and eliminating these triggers for the `dbIndexer` lambda on the Dynamo table streams. And then ultimately we can remove the `dbIndexer` lambda entirely.